content/spaces/observability.md

```diff
@@ -1,339 +1,78 @@
-The `otel-collector` gathers:
-- Kubernetes metrics
-- Crossplane, Provider, and DNS metrics
-- Scrape _any_ pods with auto scrape metrics managed by vCluster
+Upbound offers a built-in feature to help you collect and export logs, metrics, and traces for everything running in a Space. Upbound's feature an integrated observability pipeline built on the [OpenTelemetry](https://opentelemetry.io/) project. 
 
-The default configuration has an Open Telemetry trace collector enabled. vCluster
-sends traces for 10% of the total traces to prevent over-collection
-of traces and reduce noise.
+[OpenTelemetry Collectors](https://opentelemetry.io/docs/collector/) are deployed to collect, process, and expose telemetry data in Spaces. Upbound deploys a central collector at the Space-level along with collectors per control plane. By default, all collectors belonging to control planes pass their telemetry data to the Spaces collector.
 
-## Configuration
+## Usage
 
-Out of the box, Prometheus and Prometheus-compatable scrappers like
-[DataDog](https://docs.datadoghq.com/containers/kubernetes/prometheus/?tab=kubernetesadv2#metric-collection-with-prometheus-annotations) are compatible.
+Configuration of the pipeline is done at installation time. In addition to enabling the feature, you have the ability to configure:
 
-### Example Prometheus installation
+* which exporter is used
+* an `API Key` to enable a Space to write to the designated exporter.
+* whether metrics and/or traces route to the desired exporter.
 
-Most operators probably have a monitoring solution in place, and if not,
-Amazon, Azure, and Google all offer managed monitoring solutions.
+For information about which exporters are available, consult the OpenTelemetry Collector [exporter docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/README.md).
 
-The standard for monitoring Kubernetes and Kubernetes workloads is
-Prometheus. For operators who to explore the Environment and Spaces
-metrics, Upbound recommends using Prometheus as a starting point.
-Prometheus may not meet an Operators business or technology requirements.
-Upbound advises operators to consider what metrics and observability stack meet
-meet their needs before running in production.
+Below is an example for how to configure it:
 
-The recommended way to install Prometheus is through
-[the official helm charts](https://github.com/prometheus-community/helm-charts).
+{{< tabs >}}
 
-Add the Prometheus chart repository:
-```shell
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-```
+{{< tab "Up CLI" >}}
 
-Install Prometheus, Alertmanager, Prometheus node exporter, kube-state-metrics,
-and Grafana:
-```shell
-helm install \
-    -n monitoring --create-namespace \
-    prometheus prometheus-community/kube-prometheus-stack
+```bash {hl_lines="7-11"}
+helm -n upbound-system upgrade --install spaces \
```

Comment:
Should this be an `up space init` command ?

Reply:
Good catch